control/include/tcp/controller.h

```diff
@@ -54,6 +54,10 @@ class Controller {
 
   // Get statistics.
   virtual void GetStatistics(::istio::mixer_client::Statistics* stat) const = 0;
+
+  virtual const ::istio::mixer_client::Environment& environment() const = 0;
+
+  virtual std::chrono::milliseconds report_interval_ms() const = 0;
```

Comment:
Not need to expose these two.   It is not this lib job to keep track of config.  
* proxy can use Envoy timer directly.    
* proxy can make a copy of report_interval.